app/javascript/dashboard/routes/dashboard/settings/profile/Index.vue

```diff
@@ -115,10 +123,12 @@ import { mapGetters } from 'vuex';
 import { clearCookiesOnLogout } from '../../../../store/utils/api';
 import NotificationSettings from './NotificationSettings';
 import alertMixin from 'shared/mixins/alertMixin';
+import WootButton from 'components/ui/WootButton';
```

Comment:
Can you check if this line is needed ?  Since `woot-button` is globally imported in `application.js`.

Reply:
@nithindavid Done